services/docker-compose.yml

```diff
@@ -1452,6 +1455,62 @@ services:
       - default
       - interactive_services_subnet # for legacy dynamic services
 
+  # use to define fallback routes for simcore services
+  # if docker healthcheck fails, container's traefik configuration is removed
+  # leading to 404 https://github.com/traefik/traefik/issues/7842
+  #
+  # use fallback routes to return proper 503 (instead of 404)
+  # this service must be running at all times
+  traefik-configuration-placeholder:
```

Comment:
- so if i get this right, when any of the services above are unhealthy, traefik reroutes to thisone but this has not server so how does it return a 503? I am not sure about the last part.

- This service does not even define a healthcheck, does this has an impact?